tests/end2end-tests/data_gen/remove_test_mask.py

```diff
@@ -0,0 +1,57 @@
+"""
+    Copyright 2023 Contributors
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+    This removes train/val/test masks on a dataset.
+"""
+
+import dgl
+import os
+import argparse
+
+from remove_mask import print_feat_names
+
+def remove_mask(data):
```

Comment:
This method has the same name as the remove_mask class. Can we change it to remove_test_mask()?